python/scripts/hpx_invoke.py

```diff
@@ -10,16 +11,16 @@
 
 from datetime import datetime
 
-from string import letters, digits
-
-from types import StringType
+from string import digits
 
 from optparse import OptionParser
 
-if exists(join(path[0], "../hpx")):
-  path.append(join(path[0], ".."))
-if exists(join(path[0], "../share/hpx/python/hpx")):
-  path.append(join(path[0], "../share/hpx/python"))
+import sys
+
+if exists(join(sys.path[0], "../hpx")):
+  sys.path.append(join(sys.path[0], ".."))
```

Comment:
Isn't `path` defined in the `os` module, i.e. `os.path.append(join(sys.path[0], ".."))`? That should also work on both platforms, Python V2 and V3.

Reply:
This worked for python 2.7 and python 3. So I think we can keep it as it is.